cmd/agent/app/integrations.go

```diff
@@ -404,6 +415,38 @@ func pipCheck(cachePath string) error {
 	return fmt.Errorf("error executing pip check: %v", err)
 }
 
+func moveConfigurationFiles(integration string) error {
+	confFolder := config.Datadog.GetString("confd_path")
+	check := strings.TrimPrefix(integration, "datadog-")
+	confFileDest := filepath.Join(confFolder, fmt.Sprintf("%s.d", check))
+	if err := os.MkdirAll(confFileDest, os.ModeDir); err != nil {
+		return err
+	}
+
+	here, _ := executable.Folder()
+	confFileSrc := filepath.Join(here, relChecksPath, check, "data")
+	files, err := ioutil.ReadDir(confFileSrc)
+	if err != nil {
+		return err
+	}
+
+	for _, file := range files {
+		filename := file.Name()
+		// Replace existing file
+		src := filepath.Join(confFileSrc, filename)
+		dst := filepath.Join(confFileDest, filename)
+		err := os.Rename(src, dst)
+		if err != nil {
+			return err
```

Comment:
should we fail early without trying to move the other files?

Reply:
I think so, if one fail it's most likely that the others will fail right ? (such as permission error for instance)